packages/kbn-shared-ux-components/src/empty_state/empty_state_page.mdx

```diff
@@ -0,0 +1,14 @@
+**id:** sharedUX/Components/EmptyStatePage
+**slug:** /shared-ux-components/empty_state_page
+**title:** Empty State Page
+**summary:** A page to be displayed when there is no data in Elasticsearch, or no data views
+**tags:** ['shared-ux', 'component']
+**date:** 2022-04-20
+
+---
```

Comment:
I would consider following this README: https://github.com/elastic/kibana/blob/main/packages/shared-ux/button/exit_full_screen/README.mdx